chapter_preliminaries/ndarray.md

```diff
@@ -380,6 +426,11 @@ X_var[:2, :].assign(tf.ones(X_var[:2,:].shape, dtype=tf.float32) * 12)
 X_var
 ```
 
+```{.python .input}
+X_new_2 = X_new_1.at[:2, :].set(12)
```

Comment:
Missing `#@tab jax`

Reply:
The syntax has changed, please use `%%tab jax`.